daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/SResult.scala

```diff
@@ -92,14 +92,17 @@ object Question {
 sealed abstract class SResult[+Q] extends Product with Serializable
 
 object SResult {
-  final case class SResultError(err: SError) extends SResult[Nothing]
+
+  final case class SResultQuestion[Q](question: Q) extends SResult[Q]
```

Comment:
Just a small note/complaint. I would prefer code cleanups, reordering and other small refactorings to be in different PRs, as it makes the substantive changes harder to review.

Reply:
:+1: